lib/grinder_tools.dart

```diff
@@ -139,10 +139,10 @@ Future runProcessAsync(GrinderContext context, String executable,
  * A default implementation of an `init` task. This task verifies that the grind
  * script is executed from the project root.
  */
-void defaultInit(GrinderContext context) {
+void defaultInit([GrinderContext context]) {
```

Comment:
I'm considering just removing all `context` params, and making `0.7.0` a big breaking change. I don't think it'll be a burden on the users; there aren't any semantic changes in the breaking changes, just removed args.